tests/foreman/ui/test_computeresource.py

```diff
@@ -127,19 +129,25 @@ def test_create_libvirt_resource_different_names_negative(self, name):
         @Assert: A libvirt Compute Resource is not created
 
         """
+        include_list = [' ']
```

Comment:
This is not a real blocker for me. But I think that we should use the improve if necessary approach. With that said, even if additional data _may_ be required, we should improve it as needed and not use single use vars now.